alacritty/src/display/mod.rs

```diff
@@ -502,27 +499,25 @@ impl Display {
 
         Ok(Self {
             window,
-            context: ManuallyDrop::new(Replaceable::new(context)),
-            surface: ManuallyDrop::new(surface),
-            renderer: ManuallyDrop::new(renderer),
-            glyph_cache,
-            hint_state,
-            meter: Meter::new(),
             size_info,
-            ime: Ime::new(),
-            highlighted_hint: None,
-            vi_highlighted_hint: None,
-            cursor_hidden: false,
+            hint_state,
+            glyph_cache,
+            damage_tracker,
+            raw_window_handle,
```

Comment:
Descending order would be much better here. :(

Reply:
It won't be a pyramid though, because we have descending for defaults, thus it means we need ascending for regular fields so they meet at some point?

Reply:
Realistically it should always be descending. Otherwise the order has to be changed completely and that's not possible because then default would be at the top.